api/play.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
@aseebart, how does removing this file affect the static file routing? As I recall, the static routing was never checked in, is that correct?